README.md

```diff
@@ -140,3 +153,20 @@ return (
   </Homepage>
 );
 ```
+
+### SeeAlso
+
+#### Reference
+
+| Prop      | Type                                                      | Default | Required           |
+| --------- | --------------------------------------------------------- | ------- | ------------------ |
+| `as`      | ` ol`, `li`                                               | `ol`    | --                 |

```

Comment:
```suggestion
| `as`      | ` ol`, `ul`                                               | `ul`    | --                 |

```

Reply:
I'm not sure if I'm being dumb or if it's late, but it looks like we're describing wrapper here and the two option should be `ul` and `ol`.  Also, looks like the default is currently `ul`.